hw_diag/app.py

```diff
@@ -33,6 +34,10 @@ def get_app(name):
     def run_diagnostics_task():
         perform_hw_diagnostics()
 
+    # Set hostname for mdns / netbios
+    def run_hostname_task():
```

Comment:
@robputt this works, but is probably in the wrong place? 

i guess we only need to run this once on first load of the container, rather than on a schedule.

Maybe there is a better way to do it?